prime-router/docs/universal-pipeline/route.md

```diff
@@ -1,17 +1,419 @@
+   </td>
+  </tr>
+</table>
+
+
+
+### Processing Mode Code Filter
+
+The processing mode of the data indicates the sender’s intended context for the data. Options for this field are found here [CodeSystem: processingId](https://terminology.hl7.org/5.2.0/CodeSystem-v2-0103.html). The intention is to ensure the sender and receiver operate with the same data content context. Test data should only be accepted by test receivers. Production data should only be accepted by production receivers.
+
+
+<table>
+  <tr>
+   <td><strong>Topic</strong>
+   </td>
+   <td>Applies to following topics: <em>full-elr, etor-ti</em>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Operation</strong>
+   </td>
+   <td>Expressions are evaluated with AND operation
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Default</strong>
+   </td>
+   <td>See code block below
+   </td>
+  </tr>
+</table>
+
+```kotlin
+   /**
+ * Default Rule:
+ *  Must have a processing mode id of 'P'
+ */
+val processingModeFilterDefault: ReportStreamFilter = listOf(
+    "%processingId = 'P'"
+)
+```
+
+### Condition Filter
+
+Filter data based on the test identifiers. A receiver expecting flu results should only accept tests for flu. If the message contains multiple observations, some that pass the condition filter and others that do not, the condition filter will be used to identify the desired observations. Identifiers for the needed observations are added to the Endpoint which is then added to the Provenance resource.
+
+*The Translate step will review the Endpoints and remove any observations that are not identified.
+
+
+<table>
+  <tr>
+   <td><strong>Topic</strong>
+   </td>
+   <td>Applies to following topics: <em>full-elr, etor-ti, elr-elims</em>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Operation</strong>
+   </td>
+   <td>Expressions are evaluated with OR operation
+   </td>
+  </tr>
+  <tr>
+   <td><strong>Default</strong>
+   </td>
+   <td><strong>Allow All:</strong> The door is open. No filter is in place
+   </td>
+  </tr>
+</table>
+
+
+
+## Storage
+
+The `Route Function` retrieves messages from the `pdhprodstorageaccount` Azure Storage Account. Within that storage account there is a blob container named `reports` containing folders for use by the Universal Pipeline. The `Convert Function` places all messages into the route folder for retrieval by the `Route Function`. Those messages that match a receiver's filtering will then be placed in the `translate` folder for future retrieval by the `Translate Function`. Messages within the `route` folder are saved to sub-folders equaling the name of the sender.
+
+
+## Filter Reversal
+
+This is a NOT operation on the result of the filters set in the Quality Filter. The primary use is to set the filters for a secondary receiver to ingest all data not accepted by a primary receiver. This may be helpful to keep the qualityFilter setting the same for both the primary and secondary and make it easier to read the configuration.
+
+
+## Logging
```

Comment:
This section seems to borrow from 0001-universal-pipeline-filter-reporting.md? We should NOT have the same content in multiple places, so let's figure out how to resolve the redundancy (either by refactoring how the content is split up or having one document have everything and the other referencing it).